AnthroFace/rib-fracture
# Henry

## Purpose
Exporter now works correctly and the template has also been updated to actually work with the importer.

A few minor spelling errors were corrected and some small changes were made to the expected headers for imports.

#### Steps for testing (if needed)
-try exporting your database
-try re-importing to check if headers are accepted (import should give warning that case ids already exist)

## Learning
-https://github.com/react-csv/react-csv
